common/djangoapps/util/milestones_helpers.py

```diff
@@ -313,6 +314,18 @@ def remove_content_references(content_id):
     return milestones_api.remove_content_references(content_id)
 
 
+def any_unfulfilled_milestones(course_id, user_id):
+    """ Returns a boolean if user has any unfulfilled milestones """
+    from milestones.exceptions import InvalidMilestoneRelationshipTypeException
```

Comment:
This comment still needs to be addressed.  The code in this method needs to be prefixed with:

```
if not settings.FEATURES.get('MILESTONES_APP', False):
        return None
```